### PR TITLE
Docs/guide-on-using-conventional-commits-and-branches

### LINKED ISSUE #94: Write team guide on using conventional commits and branches
We are sort of using conventional commits and branches (starting to), but don't have any guides on it yet.

### PR DESCRIPTION
**Pull request by signekb**
## Description

- This PR adds guides on how to use the VS Code extensions Conventional Commits and Conventional Branch. 
- In addition, I have formatted, made some minor text edits, and moved the branch section above the commit section, since you would typically branch before committing. 

## Related Issues

Closes #94

## Reviewer Focus

This PR requires an in-depth review.

## Checklist

For general documentation:

- [X] Spell-check
- [X] Did the page(s) preview correctly on your machine without breaking
- [X] New category words (keywords) (if any) added to the code snippet file - no.

### CODE REVIEW

File: entries/workflow.qmd [hunk 1]
```diff
@@ -8,45 +8,164 @@ date: last-modified
...
+#### What are conventional branches?
+
+For naming branches, we follow naming scheme called **conventional
+branches**. Conventional branches are a way of naming branches that are
+easy to read and understand. This naming follow a specific format that
+includes a type and a branch description. Specifically, the format of a
+conventional branch name is `<type>/<branch>`, where `<type>` is the
+type of the branch and `<branch>` is a short description of the change
+that will be made. The description should be written in imperative mood
+using kebab-case.
```
Comment (K-Beicher) [inline entries/workflow.qmd:48]: Maybe a bracket at the end (eg. whitespace-replaced-with-dashes)?

**Review by K-Beicher — DISMISSED** [review]

File: entries/workflow.qmd (hunk 1 above)
Comment (signekb) [reply entries/workflow.qmd]: Good idea. I have added some examples :)

File: entries/workflow.qmd [hunk 2]
```diff
@@ -8,45 +8,164 @@ date: last-modified
...
+When creating branches and pull requests, follow these general
+guidelines:
+
+-   Create branches following the [Conventional
+    Branches](https://github.com/pshaddel/conventional-branch) approach
+    by using the VSCode Extension, that is also in our (automatically)
+    recommended extension list (see tip below).
+-   When creating branches and pull requests, keep them as focused and
+    specific to the issue as is reasonable and limit how many changes
+    are made in that Pull Requst. The fewer changes made in a pull
```
Comment (lwjohnst86) [inline entries/workflow.qmd:23]:
```suggestion
    are made in that pull request. The fewer changes made in a pull
```

to be consistent in wording.

File: entries/workflow.qmd [hunk 3]
```diff
@@ -8,45 +8,164 @@ date: last-modified
...
+using kebab-case. For example `feat/add-new-feature` or `fix/bug-fix`.
+
+#### Using the VS Code extension
+
+To ease the process of creating conventional branches, we use the VS
+Code extension "Conventional Branches" by pshaddel. This extension
+provides autocompletion for conventional branch names and can help you
+create branches that follow the conventional branches format. This
+extension is in our recommended extensions and should automatically pop
+up, when you're using VS code to edit any of the Seedcase GitHub
```
Comment (lwjohnst86) [inline entries/workflow.qmd:57]:
```suggestion
up to install when you're using VS code to edit any of the Seedcase GitHub
```

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Very nice 🌠 just some small edits 🥇

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**